data_subscriptions/notifications/ckan_metadata.py

```diff
@@ -15,7 +15,9 @@ class CKANMetadata:
     def __init__(self, action, entity_ids):
         self.action = action
         self.entity_ids = entity_ids
-        self.api = RemoteCKAN(CKAN_URL, apikey=CKAN_API_KEY)
+        self.api = RemoteCKAN(CKAN_URL,
+                              apikey=CKAN_API_KEY,
+                              user_agent="data-subscription/latest (API call for CKAN activity pull)")
```

Comment:
@sagargg I think this is for getting metadata about packages so comment in brackets would be:

`data-subscription/latest (API call for CKAN package metadata details)`